auth0/auth0-authorization-extension.js

```diff
@@ -0,0 +1,59 @@
+/*
+*  This rule been automatically generated by auth0-authz-extension
+*  Updated by kerin.cosford@digital.justice.gov.uk, 2017-01-14T22:47:58.651Z
+ */
+function (user, context, callback) {
+  // If connection is not passwordless skip this rule
+  if (context.connection !== 'email') {
+    return callback(null, user, context);
+  }
+
+  var _ = require('lodash');
+  var EXTENSION_URL = "https://kerinmoj.eu.webtask.io/adf6e2f2b84784b57522e3b19dfc9201";
+
+  getPolicy(user, context, function(err, res, data) {
+    if (err) {
+      console.log('Error from Authorization Extension:', err);
+      return callback(new UnauthorizedError('Authorization Extension: ' + err.message));
+    }
+
+    if (res.statusCode !== 200) {
+      console.log('Error from Authorization Extension:', res.body || res.statusCode);
+      return callback(
+        new UnauthorizedError('Authorization Extension: ' + ((res.body && (res.body.message || res.body) || res.statusCode)))
+      );
+    }
+
+    // Add permissions (from Authorization Extension) to the user object.
+    user.permissions = data.permissions;
```

Comment:
`data.permissions` is an array of permissions (strings) provided by the Authorisation Extension.